Dockerfile

```diff
@@ -16,8 +16,8 @@ RUN    apt-get update                      \
                        python3-pip         \
                        python3-venv
 
-ARG USER_ID=1000
```

Comment:
Note this was changed as we discovered in pyk execution there can cause overlap in the IDs and cause conflicts. Picking a random ID known to not be used unless there are 8876 additional users already on the system.

Reply:
I think instead we should be passing this in as an argument every time, and making sure it matches inside and outsidet the container. That's important because otherwise we end up leaving files on the host system that do not have the correct ownership for the GH runner to cleanup afterwards.

This is where the K Dockerfile has the argument, for example: https://github.com/runtimeverification/k/blob/master/.github/workflows/Dockerfile#L73, and where it receives the userid from the gh-runner user outside on the host: https://github.com/runtimeverification/k/blob/master/.github/actions/with-docker/action.yml#L42